pkg/logs/input/journald/tailer_util.go

```diff
@@ -0,0 +1,58 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package journald
```

Comment:
IMHO, it's more a specific journald logic (that does not provide image information in fields we receive) than docker itself. It's not meant to be used elsewhere so I put it next to client code. But I agree we could see both ways, I don't mind moving it if there is a strong preference for that.

Reply:
Up to you, in general I am not a huge fan of `utils` or `commons` because the logic is very often tied to something specific which I think should be the name of file/package but reusable in different places but this is very subjective.